officers.json

```diff
@@ -68,5 +95,411 @@
+    "term_start": "SUMMER_2015",
+    "term_end": "SPRING_2016",
+    "graduating_semester": ""
+  },
+  {
+    "name": "L Wilson",
+    "position": "VICE_PRESIDENT",
+    "description": "",
+    "github": "",
+    "slack": "",
+    "personal_site": "",
+    "image": "",
+    "special_ability": "",
+    "join_date": "",
+    "term_start": "SUMMER_2015",
+    "term_end": "SPRING_2016",
+    "graduating_semester": ""
+  },
+  {
+    "name": "Lee Zimmerman",
+    "position": "TREASURER",
+    "description": "",
+    "github": "",
+    "slack": "",
+    "personal_site": "",
+    "image": "",
+    "special_ability": "",
+    "join_date": "",
+    "term_start": "SUMMER_2015",
+    "term_end": "FALL_2016",
+    "graduating_semester": ""
+  },
+  {
+    "name": "Tony Phelps",
+    "position": "PRESIDENT",
+    "description": "",
+    "github": "",
+    "slack": "",
+    "personal_site": "",
+    "image": "",
+    "special_ability": "",
+    "join_date": "",
+    "term_start": "SUMMER_2014",
+    "term_end": "SPRING_2015",
+    "graduating_semester": ""
+  },
+  {
+    "name": "Jeff Gensler",
+    "position": "VICE_PRESIDENT",
+    "description": "",
+    "github": "",
+    "slack": "",
+    "personal_site": "",
+    "image": "",
+    "special_ability": "",
+    "join_date": "",
+    "term_start": "SUMMER_2014",
+    "term_end": "SPRING_2015",
+    "graduating_semester": ""
+  },
+  {
+    "name": "Carlos Ortega",
+    "position": "TREASURER",
+    "description": "",
+    "github": "",
+    "slack": "",
+    "personal_site": "",
+    "image": "",
+    "special_ability": "",
+    "join_date": "",
+    "term_start": "SUMMER_2014",
+    "term_end": "SPRING_2015",
+    "graduating_semester": ""
+  },
+  {
+    "name": "Sunil Rao",
+    "position": "SECRETARY",
+    "description": "",
+    "github": "",
+    "slack": "",
+    "personal_site": "",
+    "image": "",
+    "special_ability": "",
+    "join_date": "",
+    "term_start": "SUMMER_2014",
+    "term_end": "SPRING_2015",
+    "graduating_semester": ""
+  },
+  {
+    "name": "Tony Phelps",
+    "position": "PRESIDENT",
+    "description": "",
+    "github": "",
+    "slack": "",
+    "personal_site": "",
+    "image": "",
+    "special_ability": "",
+    "join_date": "",
+    "term_start": "SUMMER_2013",
+    "term_end": "SPRING_2014",
+    "graduating_semester": ""
+  },
+  {
+    "name": "Chase Lee",
+    "position": "VICE_PRESIDENT",
+    "description": "",
+    "github": "",
+    "slack": "",
+    "personal_site": "",
+    "image": "",
+    "special_ability": "",
+    "join_date": "",
+    "term_start": "SUMMER_2013",
+    "term_end": "SPRING_2014",
+    "graduating_semester": ""
+  },
+  {
+    "name": "Carlos Ortega",
+    "position": "TREASURER",
+    "description": "",
+    "github": "",
+    "slack": "",
+    "personal_site": "",
+    "image": "",
+    "special_ability": "",
+    "join_date": "",
+    "term_start": "SUMMER_2013",
+    "term_end": "SPRING_2014",
+    "graduating_semester": ""
```

Comment:
Carlos here was treasurer both from 2013-4 and 2014-5. Do we want to merge those into a single entry, or have two? I suspect we only want one, since the dates are the only thing that changes. @BluuArc , what do you think? Can the frontend handle this situation?

Reply:
The way I'm planning on doing the frontend listing of this is creating sections per semester, so multiple entries shouldn't be an issue nor is it an issue if they're combined into one entry (assuming that Carlos was in the same position for the entire combined time period).